specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json

```diff
@@ -504,12 +504,33 @@
         "ImmediatePurgeDataOn30Days": {
           "type": "boolean",
           "description": "Purge data immediately after 30 days."
+        },
+        "PrivateLinkScopedResources": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkScopedResource"
```

Comment:
since I'm also working on this feature for Azure CLI, I have a quick question for this one. Does it mean several private link scoped resource can be linked to one app component?